src/actions/auth.js

```diff
@@ -105,7 +119,39 @@ export const watchUserProfile = (dispatch, firebase) => {
                 const populates = getPopulateObjs(profileParamsToPopulate)
                 const profile = snap.val()
                 forEach(populates, (p) => {
-                  set(profile, p.child, get(data, `${p.root}.${snap.val()[p.child]}`))
+                  const child = get(profile, p.child)
```

Comment:
Nice use of get here, this will keep it so that dot notation can be used for child parameter of populate objects.